pkg/geoipupdate/geoip_updater.go

```diff
@@ -15,14 +15,26 @@ import (
 	"github.com/pkg/errors"
 )
 
+// AddHeaderRoundTripper struct is used to add versioned User-Agent header to each request.
+type AddHeaderRoundTripper struct {
+	r http.RoundTripper
+	v string
+}
+
+// RoundTrip is used to add versioned User-Agent header to each request.
+func (ahrt AddHeaderRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
+	r.Header.Add("User-Agent", "geoipupdate/"+ahrt.v)
+	return ahrt.r.RoundTrip(r)
```

Comment:
I think you can just do something like:
```suggestion
	w, err := ahrt.r.RoundTrip(r)
	if err != nil {
		return nil, errors.Wrap(err, "calling RoundTrip)
	}
	return w, nil
```

Reply:
Since we've found code with explicit err type expectations in the upstream code, should we just disable this particular lint check and not wrap the error?

https://github.com/golang/go/blob/2ebe77a2fda1ee9ff6fd9a3e08933ad1ebaea039/src/net/http/omithttp2.go#L52